src/Bicep.Cli/Arguments/GenerateParametersFileArguments.cs

```diff
@@ -82,6 +108,16 @@ public GenerateParametersFileArguments(string[] args) : base(Constants.Command.G
                 throw new CommandLineException($"The --outdir and --outfile parameters cannot both be used");
             }
 
+            if (OutputFormat is null)
+            {
+                OutputFormat = "json";
+            }
```

Comment:
[nit] can be simplified to `OutputFormat ??= "json";`

Reply:
I like it 👍 just updated the code